community_scripts.json

```diff
@@ -1391,5 +1391,21 @@
       "windows"
     ],
     "category": "TRMM (Win):Updates"
+  },
+  {
+    "guid": "ace9aecf-a466-4589-865c-54817436f194",
+    "filename": "Win_Teamviewer_Host_custom_install.ps1",
+    "submittedBy": "https://github.com/clanto007/",
+    "name": "TeamViewer Host with Custom ID and API",
```

Comment:
What about "Teamviewer - Install Host with Custom ID and API"

We're probably going to have more Teamviewer scripts...so you have a visual delineator when sorting as title